src/functional/ffront/dialect_parser.py

```diff
@@ -96,14 +96,16 @@ def apply(
         try:
             definition_ast = ast.parse(textwrap.dedent(source)).body[0]
             definition_ast = RemoveDocstrings.apply(definition_ast)
-            definition_ast = FixMissingLocations.apply(definition_ast)
-            definition_ast = ast.increment_lineno(definition_ast, starting_line - 1)
+            definition_ast_formatted = FixMissingLocations.apply(definition_ast)
```

Comment:
Please remove the changes in this file as they break #942. I applied a different fix there just now (https://github.com/GridTools/gt4py/pull/942/commits/89cf60bc232149627fda797ab58ea014479755c0).